website/public/js/controllers/authCtrl.js

```diff
@@ -46,10 +46,10 @@ angular.module('habitrpg')
 
         $scope.registrationInProgress = true;
 
-        var url = ApiUrl.get() + "/api/v2/register";
+        var url = ApiUrl.get() + "/api/v3/user/auth/local/register";
         if($rootScope.selectedLanguage) url = url + '?lang=' + $rootScope.selectedLanguage.code;
         $http.post(url, scope.registerVals).success(function(data, status, headers, config) {
-          runAuth(data.id, data.apiToken);
+          runAuth(data.data.id, data.data.apiToken);
```

Comment:
When we do the client rewrite, we'll want these to be `res` or something so it becomes `res.data`